swarm/network/stream/syncer_test.go

```diff
@@ -345,3 +318,336 @@ func TestDifferentVersionID(t *testing.T) {
 	log.Info("Simulation ended")
 
 }
+
+// Tests that when two nodes connect:
```

Comment:
I think we should be explicit that there are no other connections, but the connection between these 2 nodes.

Otherwise when 2 nodes connect it is not intuitive to expect them to sync all their chunks.